client/post-editor/editor-seo/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+
+function getSeoTabUrl( slug ) {
+	return `/settings/seo/${ slug }`;
+}
+
+export default React.createClass( {
+	displayName: 'EditorSeo',
+
+	propTypes: {
+		site: React.PropTypes.object.isRequired,
+		post: React.PropTypes.object.isRequired
+	},
+
+	render() {
+		const { slug } = this.props.site;
+
+		// Temporary placeholder chips for design review
+		const sampleChips = [ 'Post Title', 'Site Title' ];
+
+		return (
+			<Accordion
+				title={ this.translate( 'Advanced SEO' ) }
+				icon={ <Gridicon icon="search" /> }
+				className="editor-drawer__seo"
```

Comment:
`className` should be `editor-seo` to match directory in which it exists ([guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines)). 

We'll soon have an ESLint warning to help identify these issues: https://github.com/Automattic/eslint-plugin-wpcalypso/pull/7